mod.conf

```diff
@@ -1,5 +1,6 @@
 name = digilines
 depends = default
+optional_depends = tubelib,tubelib2
```

Comment:
It might be helpful to add this new feature to the README, even if it's currently very minimalistic.

Reply:
I've added a little it, the behaviour is primarily the same as pipeworks though and it seems that is covered. I've added a few sentences on the similarities and differences